.github/ISSUE_TEMPLATE/02_bug.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: A concrete bug report with steps to reproduce the behavior.
+labels: bug,
+---
+
+### Bug description
+
+
+
+### How is the bug triggered?
+How can you reproduce the bug?
+1.
+
+
+### What did you expect to happen?
+
+
+
+### How are you running the application?
```

Comment:
Would we also want to ask for the Python version and implementation?

Reply:
Some of these are listed in the About menu, so it's difficult to know whether to include these in the top part or beneath that.

The reason I included these separately is
- in case users cannot access the about menu
- if we later add a 'copy' option to the about menu for this kind of reporting :)

The python details could probably go here, but may want to be freeform - maybe multiple versions have been tested?

Reply:
oh yeah, we could mention that these details can all be found in the About menu, at the starting of that section. Since one can't assume that, if they do not know.
Nice ideas! I didn't account for these possibilities.